torchrl/data/datasets/minari_data.py

```diff
@@ -281,6 +283,7 @@ def _download_and_preproc(self):
                         f"loading dataset from local Minari cache at {h5_path}"
                     )
                     h5_data = PersistentTensorDict.from_h5(h5_path)
+                    h5_data = h5_data.to_tensordict()
```

Comment:
do we need this? It's a bit expensive so if we can avoid it it's better (under the hood it copies the entire dataset in memory)